README.md

```diff
@@ -14,20 +14,33 @@ column_name = "customer_id"
 schema_name = "public"
 database_url = "postgres://localhost:15432/postgres"
 skip_tables = [
-  "daily_exchange_rates"
+  "users_backup"
 ]
 
 [overrides]
 # We only want the one customer identified by --id on the command line
 customers = """
-  select * from customers where id = :id
+  select * from customers where id in :ids
 """
 # The `user_files` table doesn't have a customer_id column, so we need to join.
 user_files = """
   select user_files.*
   from users_files
   join users on users.id = user_files.user_id
-  where users.customer_id = :id
+  where users.customer_id in :ids
+"""
+daily_exchange_rates = """
+  select * from daily_exchange_rates
+  where ARRAY['currency'] && (current_setting('pg_parcel.features')::text[]);
+"""
+audit_log = """
+  select * from audit_log
+  where customer_id in :ids and
+  (
+    ARRAY['audit'] && (current_setting('pg_parcel.features')::text[])
```

Comment:
💅 This can also be written as:

```sql
    'audit' = any (current_setting('pg_parcel.features')::text[])
```

Slightly more concise, but not much in it.

Reply:
I went with the `array && array` syntax because I wanted to provide an example that would also cover cases where two features require the same table to be fully populated.